src/Slider/Slider.jsx

```diff
@@ -222,6 +224,10 @@ const Slider = class Slider extends React.Component {
   handleDocumentScroll() {
     if (!this.props.touchEnabled) return;
     this.isDocumentScrolling = true;
+    window.clearTimeout(this.scrollStopTimer);
```

Comment:
Can you duplicate this same line `window.clearTimeout(this.scrollStopTimer);` in componentWillUnmount() and then also set scrollStopTimer to null in componentWillUnmount()

It's a rare condition, but it will stop a memory leak.

Other than that, this looks good.

Reply:
Done